Signal/translations/en.lproj/Localizable.strings

```diff
@@ -148,7 +148,7 @@
 /* Alert title when picking a document fails because user picked a directory/bundle */
 "ATTACHMENT_PICKER_DOCUMENTS_PICKED_DIRECTORY_FAILED_ALERT_TITLE" = "Unsupported File";
 
-/* Short text label for a voice message attachment, used for thread preview and on lockscreen */
+/* Short text label for a voice message attachment, used for thread preview and on the lock screen */
 "ATTACHMENT_TYPE_VOICE_MESSAGE" = "Voice Message";
```

Comment:
FYI these comments are pulled from the source code from lines like this:

```
attachmentString = NSLocalizedString(@"ATTACHMENT_TYPE_VOICE_MESSAGE",
                @"Short text label for a voice message attachment, used for thread preview and on lockscreen");
```

So any changes to comments in the translation file will get clobbered the next time we run auto-genstrings.

I'm in the process of backporting these comment changes to their respective source files now.